### DISCUSSION TITLE
The preprocessing code for the VIPL-HR data set in the utils/utils.py file does not work with the real VIPL-HR data set

### DISCUSSION THREAD

**Discussion by BugMaker2002** [Dylan-H-Wang/SLF-RPM#12] [Q&A] [answered]
The picture below is the directory structure of the VIPL-HR data set：
![image](https://github.com/Dylan-H-Wang/SLF-RPM/assets/110211332/83841114-72ce-461f-9434-550a03a40326)
 You can see that there are many directory levels: "VIPL-HR/data/p1/v1/source1", so how should I define the dataset_path variable of the "extract_vipl_hr_dataset()" function?
And in the "extract_vipl_hr_dataset()" function, do the two lines of code "if len(video_file_path) != 5" and "video_file_path.sort()" mean that in the "VIPL-HR/data/p1/v1/source1" folder there are multiple videos in .avi format? But obviously there is only one video (_i.e._ video.avi) in .avi format in the picture above
![image](https://github.com/Dylan-H-Wang/SLF-RPM/assets/110211332/16b11c35-31cd-4bb7-b4c5-62309d700467)

I'm not sure whether I downloaded the wrong VIPL-HR dataset or the VIPL-HR dataset you used is different from mine
**I would appreciate it very much if you would answer for me ！**

**Comment by Dylan-H-Wang** [comment, marked as answer]
This code is using VIPL-HR-V2 not V1. Check README and paper.

**Reply by BugMaker2002** [reply]
Thank you very much
